src/app/homeSignIn/homeSignIn.component.js

```diff
@@ -0,0 +1,62 @@
+import angular from 'angular';
+import includes from 'lodash/includes';
+
+import commonModule from 'common/common.module';
+import loadingOverlay from 'common/components/loadingOverlay/loadingOverlay.component';
+import sessionService from 'common/services/session/session.service';
+import sessionModalService from 'common/services/session/sessionModal.service';
+
+import template from './homeSignIn.tpl';
+
+let componentName = 'homeSignIn';
+
+class HomeSignInController {
+
+  /* @ngInject */
+  constructor(sessionService, sessionModalService) {
+    this.sessionService = sessionService;
+    this.sessionModalService = sessionModalService;
+  }
+
+  $onInit() {
```

Comment:
Can you test this? I think it'd be pretty easy. You might have to mock sessionService.